### DISCUSSION TITLE
Check answers

### DISCUSSION THREAD

**Discussion by bameyrick** [ONSdigital/design-system#1392] [Feedback]
Use this place to discuss the [check answers pattern](https://ons-design-system.netlify.com/patterns/check-answers) in the [ONS Design System](https://ons-design-system.netlify.com)

**Comment by jrbarnes9** [comment]
We should include guidance on playing back user data as they entered it, in the "Play back of a single answer" section of the check answers pattern. For example, playback a users phone number the way it was entered, don't automatically format it into a pattern.

**Reply by jrbarnes9** [reply]
#1574 raised

**Comment by liammcmurray** [comment]
Looks like images are broken on the page currently - https://service-manual.ons.gov.uk/design-system/patterns/check-answers
